core/storage/feedback/gae_models.py

```diff
@@ -183,13 +184,16 @@ def get(cls, exploration_id, thread_id, message_id, strict=True):
         return super(FeedbackMessageModel, cls).get(instance_id, strict=strict)
 
     @classmethod
-    def get_messages(cls, exploration_id, thread_id):
+    def get_messages(cls, exploration_id, thread_id, is_full_thread_id=False):
```

Comment:
Oh, goodness, no, please don't do this. We are going to end up with a whole lot of confusion about what's the full thread id and what's the thread id.

Keep the API as is, and do any conversions needed before calling it.